benchmark/src/main/scala/kantan/csv/benchmark/Decoding.scala

```diff
@@ -110,6 +110,7 @@ object Decoding {
     new CsvIterator(parser)(_.parseNext()).toList
   }
 
+  @SuppressWarnings(Array("org.wartremover.warts.SeqApply"))
```

Comment:
Fix
```
[error] .../scala/kantan/kantan.csv/benchmark/src/main/scala/kantan/csv/benchmark/Decoding.scala:135:62: [wartremover:SeqApply] Seq.apply is disabled
[error]     CSVReader.open(new StringReader(str)).iterator.map(r => (r(0).toInt, r(1), r(2).toBoolean, r(3).toFloat)).toList
[error]                                                              ^

```

Reply:
But do we now why `Seq.apply` is disabled? And if we disagree with it, shouldn't this be disabled *everywhere*, so that we don't have to manually disable it everywhere it's called?

Reply:
In general we disagree with this usage but this line is an exception...

Personally, I don't like much Warts remover for this exact matter.

Reply:
I mean why does wartremover think `Seq.apply` is a bad idea, but yeah, now that I see the code it's obvious.
Arguably, yes, this should be a for comprehension of `r.lift(index)`...

It's in the benchmarks so I don't particularly care. If you're happy with manually disabling it locally, then that's fine.

Oh and yes, the list of warts comes from kantan.sbt, it's just: you upgraded wartremover, so new warts that I never encountered were written, and I haven't had the chance to disable them because I disagreed with them.